src/formatDate.js

```diff
@@ -8,7 +8,44 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat[fromFormat.length - 1];
+  const toSeparator = toFormat[toFormat.length - 1];
```

Comment:
The logic for determining the separators assumes that the last element of the format arrays is the separator. This might not be a valid assumption if the format arrays do not include the separator as their last element. Consider revising this logic to handle cases where the separator is not explicitly included in the format arrays.